addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties

```diff
@@ -247,6 +247,6 @@ ascanbeta.usernameenumeration.alert.attack = Manipulate [{0}] field: [{1}] and m
 ascanbeta.usernameenumeration.alert.extrainfo = [{0}] parameter [{1}] leaks information on whether a user exists. The [{5}] differences in output, for the valid original username value [{2}], and invalid username value [{3}] are:\n[{4}]
 ascanbeta.usernameenumeration.desc = It may be possible to enumerate usernames, based on differing HTTP responses when valid and invalid usernames are provided. This would greatly increase the probability of success of password brute-forcing attacks against the system. Note that false positives may sometimes be minimised by increasing the 'Attack Strength' Option in ZAP.  Please manually check the 'Other Info' field to confirm if this is actually an issue. 
 ascanbeta.usernameenumeration.name = Possible Username Enumeration
-ascanbeta.usernameenumeration.refs = https://owasp.org/www-project-web-security-testing-guide/v41/4-Web_Application_Security_Testing/03-Identity_Management_Testing/04-Testing_for_Account_Enumeration_and_Guessable_User_Account.html\nhttp://sebastian-schinzel.de/_download/ifip-sec2011.pdf
```

Comment:
Use `https://dl.ifip.org/db/conf/sec/sec2011/FreilingS11.pdf` rather than remove?

Reply:
resolved in the latest commit